davidshare/Fast-Food-Fast
# Merge user interface feature branch with develop branch

#### What does this PR do?
gh-pages branch with develop branch

#### Description of Task to be completed?
- merge templates for user interface to the develop branch

#### How should this be manually tested?
- clone the repo and navigate to the root directory using the command line
- go to the ui directory in the root directory and open all html files to see UI templates
